distributedsocialnetwork/api/views.py

```diff
@@ -772,6 +772,21 @@ def post(self, request):
             "success": False,
             "message": ("Must be of type application/json. Type was " + str(request.headers["Content-Type"]))}, status=status.HTTP_400_BAD_REQUEST)
 
+
+# ====== /api/author ======
+class Authors(APIView):
+    def get(self, request):
+        if not request.user.is_authenticated:
+            return Response({
+                "query": "author",
+                "success": False,
+                "message": "Authentication is required for this endpoint."
+            }, status=status.HTTP_401_UNAUTHORIZED)
+
+        authors = Author.objects.filter(is_node=False, is_staff=False)
```

Comment:
May want to add `host=settings.FORMATTED_HOST_NAME` to the filter so that we don't return any foreign authors via the endpoint.

Reply:
good catch! done.